### PR TITLE
1156 test scaling

### PR DESCRIPTION
**Pull request by IngridKJ**
## Proposed changes

This PR addresses #1156 and concerns testing of scaling in the model classes in PorePy (fluid mass balance, energy balance, momentum balance etc.). Specifically, the tests that are altered are the test_unit_conversion tests. They are now using nontrivial material parameter values. The values are stored in fluid_values.py and solid_values.py in the dictionaries "extended_X_values_for_testing", where X is either water or granite.

## Types of changes

What types of changes does this PR introduce to PorePy?
- [ ] Minor change (e.g., dependency bumps, broken links).
- [ ] Bugfix (non-breaking change which fixes an issue).
- [ ] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [x] Testing (contribution related to testing of existing or new functionality).
- [ ] Documentation (contribution related to adding, improving, or fixing documentation).
- [ ] Maintenance (e.g., improve logic and performance, remove obsolete code).
- [ ] Other:

## Checklist

- [x] The documentation is up-to-date.
- [x] Static typing is included in the update.
- [x] This PR does not duplicate existing functionality.
- [x] The update is covered by the test suite (including tests added in the PR).
- [ ] If new skipped tests have been introduced in this PR, `pytest` was run with the `--run-skipped` flag.

### CODE REVIEW

File: tests/models/test_fluid_mass_balance.py [hunk 1]
```diff
@@ -415,10 +415,24 @@ def test_unit_conversion(units):
     class Model(SquareDomainOrthogonalFractures, SinglePhaseFlow):
         """Single phase flow model in a domain with two intersecting fractures."""
 
+        def bc_values_pressure(self, boundary_grid: pp.BoundaryGrid) -> np.ndarray:
+            """Ensure nontrivial solution."""
+            vals = self.fluid.pressure() * np.ones(boundary_grid.num_cells)
+            faces = self.domain_boundary_sides(boundary_grid).east
+            vals[faces] += self.fluid.convert_units(1e10, "Pa")
```
Comment (IvarStefansson) [inline tests/models/test_fluid_mass_balance.py:422]: Isn't this a somewhat extreme value?

**Review by IvarStefansson — APPROVED** [review]
Looks good. Thanks for the contribution!

File: tests/models/test_fluid_mass_balance.py (hunk 1 above)
Comment (IngridKJ) [reply tests/models/test_fluid_mass_balance.py:422]: Heh, yeah it is.. I was playing around with values to check the solution. Forgot to change back to something more meaningful (e.g. 1e5 I assume). Should it be changed back?

Comment (IvarStefansson) [reply tests/models/test_fluid_mass_balance.py:422]: Yes, please.

**Pull request merged**